docs/apm/traces/tracing-dashboards.md

```diff
@@ -19,13 +19,13 @@ Services must be active within the last 15 minutes to appear on the list. Recen
 
 Each dashboard is a fully customizable set of panels based on automatically generated metrics using tracing data as input. You can get insight into microservice health by looking at stats from:
 
-* **Latency**: (real-time average) the average time it takes for entry spans for that service to complete. Apart from average you can also select p99, p95, p90, p50 options in service_latency_type filter to view other percentile aggregations.<br/> ![service-latency.png](/img/traces/service-latency.png)
-* **Requests**: (real-time counter) the number of entry spans reported by the service
-* **Errors**: (real-time counter) the number of entry spans for the service that finished with an error
+* **Latency**: (real-time average) the average time it takes for entry spans for that service to complete. Apart from average you can also select p99, p95, p90, p50 options in the `latency_type` filter to view other percentile aggregations. Percentile aggregations are based on the [metrics histograms](/docs/metrics/introduction/metric-histograms/) feature.<br/> ![latency-type.png](/img/traces/latency-type.png)
```

Comment:
**Latency**.

Reply:
The punctuation here is correct, so it can go.